retriever/__main__.py

```diff
@@ -22,10 +22,12 @@
 if hasattr(sys, 'setdefaultencoding'):
     sys.setdefaultencoding(encoding)
 
-from retriever import VERSION, SCRIPT_LIST, HOME_DIR, sample_script, CITATION
-from retriever.engines import engine_list
+from retriever import sample_script, CITATION
+from retriever.lib.defaults import VERSION, HOME_DIR
```

Comment:
VERSION in` from retriever.lib.defaults import VERSION, HOME_DIR` not being used